### PR TITLE
Fix rich text editor content not being saved

### PR DESCRIPTION
**Pull request by jwm0**
I want to merge this change because it fixes major regression which causes all rich text editors to not save their content.

**PR intended to be tested with API branch:** 

### Pull Request Checklist

1. [ ] This code contains UI changes
2. [ ] All visible strings are translated with proper context including data-formatting
3. [ ] Attributes `[data-test-id]` are added for new elements
4. [ ] Changes are mentioned in the changelog
5. [ ] The changes are tested in different browsers and in light/dark mode

### Test environment config

API_URI=https://qa.staging.saleor.cloud/graphql/

### CODE REVIEW

**Comment by patrys** [comment]
Task linked: [SALEOR-3728 Rich text editor changes are not saved](https://app.clickup.com/t/2549495/SALEOR-3728)

**Review by eCzerniak — APPROVED** [review]

File: src/components/Attributes/AttributeRow.tsx [hunk 1]
```diff
@@ -160,6 +162,7 @@ const AttributeRow: React.FC<AttributeRowProps> = ({
       return (
         <BasicAttributeRow label={attribute.label}>
           <RichTextEditor
+            key={entityId} // temporary workaround, TODO: refactor rich text editor
```
Comment (orzechdev) [inline src/components/Attributes/AttributeRow.tsx:165]: I don't see any usage of `key` in `RichTextEditor` component and `key` is also reserved prop for React lists, so how does it solve the problem?

Comment (patrys) [reply src/components/Attributes/AttributeRow.tsx:165]: It's an ugly workaround that causes react to rerender the component (and lose part of its state like scroll position).

**Review by orzechdev — APPROVED** [review]
Ok, I got it 👍

**Pull request merged**